apps/expo/package.json

```
@@ -5,10 +5,10 @@
   "main": "index.js",
   "scripts": {
     "android": "expo start --android",
-    "android:dev": "concurrently \"RECHUNK_ENVIRONMENT=dev expo start --android\" \"rechunk dev-server\"",
+    "android:dev": "concurrently  \"rechunk dev-server\" \"RECHUNK_ENVIRONMENT=dev expo start --android -c\"",
```

🤔 I see you've added it for `ios`, so we can add it for `android` as well

```suggestion
    "android:dev": "concurrently  \"rechunk dev-server\" \"RECHUNK_ENVIRONMENT=dev pnpm expo start --android -c\"",
```

---

fixed :)